### DISCUSSION TITLE
I get nothing from this PDF. No Text or Table

### DISCUSSION THREAD

**Discussion by robbanp** [jsvine/pdfplumber#1124] [Ask for help with specific PDFs]
I'm not sure why, if it's the settings or data. I tried with multiple pdfs from the same source.
[tst.pdf](https://github.com/jsvine/pdfplumber/files/14967439/tst.pdf)

import pdfplumber   
print(pdfplumber.__version__)
pdf = pdfplumber.open("../data/tst.pdf")
p0 = pdf.pages[1]
p0.extract_words()

returns []

...

im = p0.to_image(resolution=75)
im

returns
![image](https://github.com/jsvine/pdfplumber/assets/2025820/fd3f59f3-7bf3-4b97-ab80-4863a00e189a)

p0.extract_table()

returns

[['', '', '', '', '', '', '', ''],
 ['', '', '', '', '', '', '', ''],
 [None, '', None, None, None, None, None, None],
 ['', '', '', '', '', '', '', ''],
 [None, '', None, None, None, None, None, None],
 ['', '', None, None, None, None, None, None],
 [None, '', None, None, None, None, None, None],
 ['', '', '', '', '', '', '', ''],
 [None, '', None, None, None, None, None, None],
 ['', '', '', '', '', '', '', ''],
 [None, '', None, None, None, None, None, None]]

**Comment by jsvine** [comment]
It appears that the text in that PDF is rasterized (converted from actual information about the characters to an image version of them). You can test this by opening the PDF in standard PDF-viewing software and attempting to select or copy-paste the text. 

To resolve the problem, you can try running the PDF through optical character recognition (OCR) software, but be aware that `pdfplumber` does not work as well with OCR'ed PDFs.
